source/catalog.html

```diff
@@ -42,133 +41,134 @@
   <main class="catalog-main">
     <section class="catalog-page">
       <div class="catalog-page__wrapper">
-        <h1 class="catalog-page__heading">
+        <h1 class="catalog-page__heading heading">
           Каталог продукции
         </h1>
+        <h2 class="visually-hidden">Корм для котов</h2>
         <ul class="catalog-page__list">
           <li class="catalog-page__card">
-            <a class="catalog-page__link" href="#">
+            <a class="catalog-page__image-link" href="#">
               <img class="catalog-page__image" src="img/catalog/pro_orange500.png" width="140" height="112"
                 alt="Catenergy Курица 500g.">
             </a>
-            <a class="catalog-page__link" href="#">
-              <h2 class="catalog-page__card-title">Cat Energy PRO 500 г</h2>
+            <a class="catalog-page__title-link" href="#">
+              <h3 class="catalog-page__card-title">Cat Energy PRO 500 г</h3>
             </a>
-            <dl class="catalog-page__inner-list inner-list">
-              <dt class="inner-list__title">Масса</dt>
-              <dd class="inner-list__description">500 г</dd>
-              <dt class="inner-list__title">Вкус</dt>
-              <dd class="inner-list__description">Курица</dd>
-              <dt class="inner-list__title">Цена</dt>
-              <dd class="inner-list__description">700 Р.</dd>
+            <dl class="catalog-page__specifications-list specifications-list">
```

Comment:
Можно быть и попроще )
```suggestion
            <dl class="catalog-page__specifications specifications-list">
```

Или даже
```suggestion
            <dl class="catalog-page__specifications specifications">
```